tests/jeos/openstack_check_boot.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright 2019 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: JeOS OpenStack image validation
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    my $reg_code = get_var('SCC_REGCODE');
```

Comment:
Why regcode is needed here ?

Reply:
Not needed, will remove it. 
Left over, as I was trying to test scc registration, but didn't want to over complicate it for the first interation and kept the `uname -a` command :)